Address/GB.ts

```diff
@@ -0,0 +1,24 @@
+import { isly } from "isly"
+
+export type GB = {
+	countryCode: "GB"
+	street: string
+	building: string
+	zipCode: string
+	city: string
+}
+
+export namespace GB {
+	export const type = isly.object<GB>(
+		{
+			countryCode: isly.string<"GB">("GB"),
+			street: isly.string(),
+			building: isly.string(),
+			zipCode: isly.string(),
+			city: isly.string(),
+		},
+		"isoly.Address.GB"
+	)
+	export const is = type.is
```

Comment:
i think we would prefer to no longer export the is and flaw functions individually, but maybe we want some backwards compatability in this case?

Reply:
It's nice to have the is and flaw function directly on the namespace IMO.

Reply:
Yes, let's keep on reexporting them. The total code is still less given more uses, in particular the code that uses is more work maintaining than a file defining an interface.